### ISSUE TITLE
Quarkus Tools brakes Intellijs syntax highlighting

### ISSUE THREAD

**Issue by Govanni** [redhat-developer/intellij-quarkus#677]
The latest version of the plugin (1.13.1.283) breaks the syntax highlighting in Intellij.
It does not highlight usages of variables/methods/... not in the file and not in the error-stripe on the right.

Issue occurs on IntelliJ version IU-222.4345.15

Reverting the plugin back to version 1.12.0.262 'resolves' the issue.

**Comment by jeffmaury** [comment]
Can you describe what do you mean by highlight usages of variables/methods as I can see the method usage code vision.

**Comment by Govanni** [comment]
Sure, I've attached two screenshots. In both cases I selected the 'station' variable in the 'generate' method. 
With the old plugin, I can see that all the usages of that variable are highligted by IntelliJ. I aslo see a green mark on the right of each usage.

The new plugin doesn't do both.

Old plugin version:
![Old plugin](https://user-images.githubusercontent.com/8862031/200268965-8f351385-281a-45fe-9468-2985857955a8.PNG)

New plugin version:
![New plugin](https://user-images.githubusercontent.com/8862031/200268938-fb00575c-57e5-485a-afe7-f0138e29c443.PNG)
